src/cli/commands/utils.py

```
@@ -142,8 +142,10 @@ def parse_key_value_pairs(ctx, param, value):
 def load_env_file_if_exist(file):
     additional_vars = {}
     if file is not None:
-        if os.path.exists(file):
-            with open(file, "r") as file:
+        original_dir = os.environ.get("ORIGINAL_DIR", os.getcwd())
+        input_path = os.path.normpath(os.path.join(original_dir, file))
+        if os.path.exists(input_path):
+            with open(input_path, "r") as file:
```

Pylint (3.3.7)

[refactor] Redefining argument with the local name 'file'

(R1704)